en/frames_plane/reptile_dragon_2.md

```diff
@@ -0,0 +1,391 @@
+
+The other end of the cable was terminated to a Dupont connector to connect to the standard spaced headers on the ELRS RX.
+The ELRS RX was connected to the cable, and then heatshrink was used to secure the two together. 
+
+![ExpressLRS RX installed in the RD2 airframe](../../assets/airframes/fw/reptile_dragon_2/elrs_pitotstatic.jpg)
+
+A thin radio antenna tube was pushed through the top of the airframe used to mount one of the two ELRS diversity antennas upright.
+The second diversity antenna was taped to the sidewall of the airframe, 90 degrees from the alignment of the first antenna.
+The ELRS RX was attached to the sidewall of the airframe next to the airspeed pressure sensor, using double-sided tape.
+
+### USB
+
+A right angle USB C extension cable was used to allow easy access to the USB C port on the FMU.
+
+![Rear USB cable hatch](../../assets/airframes/fw/reptile_dragon_2/usb_hatch.jpg)
+
+The cable was installed such that it escapes the pixhawk heading towards the aft of the airplane. The cable continues to run to the rear hatch, where the excess length can be securely wound into a knot.
+Access to this cable can be accomplished by simply removing the rear hatch and unknotting the cable.
+
+## Software Setup
+
+For this build, we need to use the PX4 modules [crsf_rc](../modules/modules_driver.md#crsf-rc) and [msp_osd](../modules/modules_driver.md#msp-osd).
+These require some custom configuration to enable.
+
+First, follow [this guide to setup a development environment](../dev_setup/dev_env.md ) and [this guide to get the PX4 source code](../dev_setup/building_px4.md).
+
+Once a build environment has been setup, open a terminal and `cd` into the `PX4-Autopilot` directory.
+To launch the [PX4 board config tool](../hardware/porting_guide_config.md#px4-menuconfig-setup) run:
+
+```
+make ark_fmu-v6x_default boardconfig`
+```
+
+### `crsf_rc` Module
+
+PX4 includes a standalone CRSF parser module which supports telemetry and CRSF LinkStatistics.
+To use this module, the default `rc_input` module must be disabled and the `crsf_rc` module must be enabled.
+
+1. In the PX4 board config tool, navigate to the `drivers` submenu, then scroll down to highlight `rc_input`.
+2. Use the enter key to remove the `*` from `rc_input` checkbox.
+3. Scroll to highlight the `RC` submenu, then press enter to open it.
+4. Scroll to highlight `crsf_rc` and press enter to enable it.
+5. Save and exit the PX4 board config tool.
+
+### `msp_osd` Module
+
+The `msp_osd` module steams MSP telemetry to a selected serial port.
+The Caddx Vista Air Unit supports listening to MSP telemetry and will show the received telemetry values in its OSD (on screen display).
+
+1. In the PX4 board config tool, navigate to the `drivers` submenu, then scroll down to highlight `OSD`.
+2. Use the enter key to open the `OSD` submenu
+3. Scroll down to highlight `msp_osd` and press enter to enable it

```

Comment:
Don't we have to rebuild PX4 after doing the config? Then install it - as a custom build? Shouldn't we explain that?

Reply:
Yes, I will make sure this reflects that the custom build needs to be uploaded to the board too.